### ISSUE TITLE
[Front/Back] Updated Encryption Format

### ISSUE THREAD

**Issue by its-colby** [autonomys/auto-drive#83]
1. Every user has pub/priv key

**Comment by clostao** [comment]
Wouldn't this require a key pair for every user?

**Comment by its-colby** [comment]
Yes, it would. We should. Sorry, this shouldn't have been in the TODO just yet. This approach should be confirmed with @jfrank-summit  before moving forward. @clostao 

This approach is meant to enhance the UX for the sharing of encrypted files
